yt_dlp/extractor/playsuisse.py

```diff
@@ -0,0 +1,121 @@
+            'md5': '0d716b7a16c3e6ab784ef817ee9a20c1',
+            'info_dict': {
+                'id': '763211',
+                'ext': 'mp4',
+                'title': 'Knochen',
+                'description': 'md5:8ea7a8076ba000cd9e8bc132fd0afdd8'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/808675/0',
+            'md5': '7c59b60aadd84b3e36d46dea01125442',
+            'info_dict': {
+                'id': '808675',
+                'ext': 'mp4',
+                'title': 'Der Läufer',
+                'description': 'md5:9f61265c7e6dcc3e046137a792b275fd'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/817193/0',
+            'md5': 'eff6791c38784543f6d87a58bfe5de15',
+            'info_dict': {
+                'id': '817193',
+                'ext': 'mp4',
+                'title': 'Die Einweihungsparty',
+                'series': 'Nr. 47',
+                'season_number': 1,
+                'episode': 'Die Einweihungsparty',
+                'episode_number': 1,
+                'description': 'md5:91ebf04d3a42cb3ab70666acf750a930'
+            }
+        }
+    ]
+
+    _GRAPHQL_QUERY = '''
+        query AssetWatch($assetId: ID!) {
+            asset(assetId: $assetId) {
+                name
+                description
+                duration
+                episodeNumber
+                medias {
+                    type
+                    url
+                }
+                thumbnail16x9 {
+                    ...ImageDetails
+                }
+                thumbnail2x3 {
+                    ...ImageDetails
+                }
+                thumbnail16x9WithTitle {
+                    ...ImageDetails
+                }
+                thumbnail2x3WithTitle {
+                    ...ImageDetails
+                }
+            }
+        }
+        fragment ImageDetails on Image {
+            id
+            url
+        }'''
+
+    def _get_media_data(self, media_id):
+        # NOTE In the web app, the "locale" header is used to switch between languages,
+        # However this doesn't seem to take effect when passing the header here.
+        response = self._download_json(
+            'https://4bbepzm4ef.execute-api.eu-central-1.amazonaws.com/prod/graphql',
+            media_id, data=json.dumps({
+                'operationName': 'AssetWatch',
+                'query': self._GRAPHQL_QUERY,
+                'variables': {'assetId': media_id}
+            }).encode('utf-8'),
+            headers={'Content-Type': 'application/json', 'locale': 'de'})
+
+        return response['data']['asset']
+
+    def _real_extract(self, url):
+        media_id = self._match_id(url)
+        media_data = self._get_media_data(media_id)
+
+        thumbnails = [{
+            'id': thumb['id'],
+            'url': thumb['url']
+        } for key, thumb in media_data.items() if key.startswith('thumbnail') and thumb is not None]
```

Comment:
```suggestion
        thumbnails = [{
            'id': thumb.get('id'),
            'url': thumb['url']
        } for thumb in traverse_obj(media_data, lambda k, v: k.startswith('thumbnail') and v['url'])]
```

why did you revert this!?

Reply:
Because I tested it and this part failed. It returns `None` somewhere, it's not like this check wasn't there for a reason...

Reply:
the check is implied in `traverse_obj`.

test with this suggestion after merging with master (there has been some change to traverse_obj since the the PR) and if there is an issue, let me know exactly what it is